01-basics/10-create-app/script.js

```diff
@@ -1 +1,19 @@
-import { defineComponent, createApp } from 'vue'
+import { defineComponent, createApp } from 'vue/dist/vue.esm-bundler.js'
+
+const App = defineComponent({
+  name: 'App',
+
+  setup() {
+    function formatAsLocalDate(timestamp) {
+      return new Date(timestamp).toLocaleString(navigator.language, {
+        dateStyle: 'long',
+      })
+    }
+
+    return { formatAsLocalDate }
+  },
+
+  template: `<div>Сегодня {{formatAsLocalDate(new Date())}}</div>`,
```

Comment:
Text interpolation is always used with spaces: `{{ formatAsLocalDate(new Date()) }}`

Reply:
Same with the next problem